source/develop/release-management/features/storage/live-merge.html.md

```diff
@@ -19,30 +20,28 @@ Live merge makes it possible to delete virtual machine disk snapshots that are n
+*   Name: Adam Litke (<alitke@redhat.com>)
+*   Name: Greg Padgett (<gpadgett@redhat.com>)
 
 ## Current status
 
-Vdsm side completed and merge into 3.5 branch. Engine side has a few minor outstanding patches pending master and a backport to 3.5. The feature depends on libvirt and qemu features that are only available from the versions in the fedora virt-preview repo.
-
-*   Last updated on -- by (WIKI)
+Released in oVirt 3.5.1
 
 ## Detailed Description
 
 The user presentation of the live merge feature is simple - the only noticeable change is that the 'Remove' snapshot command is no longer disabled when a virtual machine is running. Behind the scenes, however, the flow is quite complex. Live merge is an asynchronous operation, and maintaining the consistency of management meta data and actual virtual machine storage requires delicate handling of many different error scenarios. The following is a detailed description of the flow and processes that need to be in place on both the engine and on VDSM. See the flow diagram below for a graphical overview of the processes.
 
-A live merge is initiated when a user clicks the 'Remove' command associated with a disk snapshot. If the virtual machine is down, the existing col merge flow is followed. If the virtual machine is up, a live merge is attempted. The engine calls the VDSM merge() verb on the host on which the virtual machine is running. First, VDSM associates a new blockJob record with the virtual machine. This record is initially populated with placeholder values so that it can serve as a record that a job is just beginning. Next, VDSM calls the libvirt virDomainBlockCommit API to start the merge. This libvirt call is asynchronous and its completion is signaled by an event (ignored by VDSM) and the job being no longer reported by the virDomainBlockJobInfo API. Finally, VDSM returns success to engine. Some considerations regarding race conditions with reporting block job information via the virtual machine Stats mechanism will appear later in this page.
+A live merge is initiated when a user clicks the 'Remove' command associated with a disk snapshot. If the virtual machine is down, the existing col merge flow is followed. If the virtual machine is up, a live merge is attempted. The engine calls the VDSM merge() verb on the host on which the virtual machine is running. First, VDSM associates a new `blockJob` record with the virtual machine. This record is initially populated with placeholder values so that it can serve as a record that a job is just beginning. Next, VDSM calls the libvirt `virDomainBlockCommit` API to start the merge. This libvirt call is asynchronous and its completion is signaled by an event (ignored by VDSM) and the job being no longer reported by the `virDomainBlockJobInfo` API. Finally, VDSM returns success to engine. Some considerations regarding race conditions with reporting block job information via the virtual machine Stats mechanism will appear later in this page.
```

Comment:
Good catch, thanks. Will post an updated series when I get back home and have more reliable wifi.